src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
It seems like you're trying to link a SCSS file in your HTML. However, browsers cannot interpret SCSS files, they only understand CSS. You should compile your SCSS to CSS and then link the resulting CSS file here.